digital-objects/omap/16-lung-codex/v1.1/raw/omap-16-lung-cell-codex.csv

```diff
@@ -0,0 +1,45 @@
+OMAP-16: Organ Mapping Antibody Panel (OMAP) for Multiplexed Antibody-Based Imaging of Lung with CODEX,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,
+Author Name(s):,Jeffrey M. Purkerson; Gloria S. Pryhuber,,,,,,,,,,,,,,,,,,,,,,,,
+Author ORCiD(s):,0000-0001-7636-4992; 0000-0002-9185-3994,,,,,,,,,,,,,,,,,,,,,,,,
+Reviewer(s):,Elizabeth McDonough; Diane C. Saunders; Ellen M. Quardokus,,,,,,,,,,,,,,,,,,,,,,,,
+Reviewer ORCiD(s):,0000-0001-7524-8260; 0000-0002-8849-6746; 0000-0001-7655-4833,,,,,,,,,,,,,,,,,,,,,,,,
+General Publication(s):,https://doi.org/10.1038/s41596-021-00556-8,,,,,,,,,,,,,,,,,,,,,,,,
+Data DOI:,https://doi.org/10.48539/HBM888.LNHN.284,,,,,,,,,,,,,,,,,,,,,,,,
+Date:,"June 15, 2024",,,,,,,,,,,,,,,,,,,,,,,,

```

Comment:
Need to update all internal dates and version numbers in all .csv and .xlsx files. Double check that publications inside the .csv also are in the metadata.yaml files.